exercises/nodejs/src/game.js

```diff
@@ -1,148 +1,154 @@
 import generator from 'random-seed'
 
 var Game = function () {
-  var players = new Array()
-  var places = new Array(6)
-  var purses = new Array(6)
-  var inPenaltyBox = new Array(6)
-
-  var popQuestions = new Array()
-  var scienceQuestions = new Array()
-  var sportsQuestions = new Array()
-  var rockQuestions = new Array()
-
-  var currentPlayer = 0
-  var isGettingOutOfPenaltyBox = false
-
-  var didPlayerWin = function () {
-    return !(purses[currentPlayer] == 6)
+  let playerNames = new Array()
+  let places = new Array(6)
+  let purses = new Array(6)
+  let inPenaltyBox = new Array(6)
+  const numberOfFields = 12;
```

Comment:
Good use of Magic number refactoring, another suggestion could be to link the places and numberOfPlaces rather than Fields

Reply:
Applied the change. Good spot, thanks for that!